gallery/workbook/microsoft.documentdb-databaseaccounts.json

```diff
@@ -10,6 +10,11 @@
 					"id": "Workbooks/CosmosDb/Resource Insights",
 					"author": "Microsoft",
 					"name": "Azure Cosmos DB Resource Insights"
+				}, 
+				{
+					"id": "Workbooks/CosmosDb/Optimize the API for MongoDB cost and performance-tsg",
+					"author": "Microsoft",
+					"name": "Optimize my API for MongoDB cost and performance"
```

Comment:
should this long text be the description? the names are usually short things with a longer description?

Reply:
i believe this is the name of the workbook